docs/options.md

```diff
@@ -781,28 +780,37 @@ Platform-specific environment variables are also available:<br/>
     In configuration mode, you can use an inline array, and the items will be joined with `&&`.
 
 
-### CIBW_MANYLINUX_*_IMAGE {: #manylinux-image}
-> Specify alternative manylinux Docker images
+### `CIBW_MANYLINUX_*_IMAGE`, `CIBW_MUSLLINUX_*_IMAGE` {: #linux-image}
```

Comment:
Let's add a dummy node that has the old `id` so that hyperlinks to #manylinux-image still work

```suggestion
<div class="link-target" id="manylinux-image"></div>

### `CIBW_MANYLINUX_*_IMAGE`, `CIBW_MUSLLINUX_*_IMAGE` {: #linux-image}
```